qemu/tests/cfg/virtio_fs_queue_size.cfg

```diff
@@ -0,0 +1,63 @@
+- virtio_fs_queue_size:
+    no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1
+    no Win2008 Win7
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
+    type = virtio_fs_share_data
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
+        required_qemu = [5.2.0,)
+    kill_vm = yes
+    start_vm = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    fs_source_type = mount
+    fs_source_dir = virtio_fs_test/
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    share_mem = yes
+    test_file = 'test_file'
+    fs_dest = '/mnt/${fs_target}'
+    vm_mem_share = yes
+    vm_mem_backend = memory-backend-file
+    vm_mem_backend_path = /dev/shm
+    driver_name = viofs
+    fs_binary_extra_options = ''
```

Comment:
```suggestion
```
Looks like we don't have to assign an empty string here.
Maybe you have other concerns? Please let me know. Thanks.

Reply:
You're right, deleted it already. Thanks.